pkg/base1/cockpit.css

```diff
@@ -78,13 +78,13 @@ a {
 /* Break up sidebar in columns in smaller sizes*/
 
 @media (min-width: 992px) {
-    .cockpit-info-table-container .cockpit-info-table {
+    .info-table-ct-container .info-table-ct {
```

Comment:
The order of ct in the class info-table-ct-container feels inconsistent with ct being applied to the end everywhere else. I understand it's made to keep consistent with info-table-ct though.
If the class name was the other way around, to .ct-info-table and .ct-info-table-container it would be a more logical order. This would also be more consistent with the order used by the icons that are prefixed, such as "fa-plus" and "pficon-edit"